modules/best-practices-for-loki-query-performance.adoc

```diff
@@ -0,0 +1,31 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-09-25
+:_mod-docs-content-type: CONCEPT
+
+[id="best-practices-for-loki-query-performance_{context}"]
+= Best practices for Loki query performance
+
+You can take the following steps to improve Loki query performance:
+
+* Ensure that you are running the latest version of the {loki-op}.
+
+* Ensure that you have migrated LokiStack schema to the `v13` version.
+
+* Ensure that you use reliable and fast object storage. Loki is a database capable of doing a tremendous amount of work on object storage.
++
+To better understand the utilization of object storage by Loki, you can use the following query in the *Metrics* dashboard in the {ocp-product-title} web console:
++
+[source]
+----
+sum by(status, container, operation) (label_replace(rate(loki_s3_request_duration_seconds_count{namespace="openshift-logging"}[5m]), "status", "${1}xx", "status_code", "([0-9]).."))
+----
+
+* Ensure that automatic stream sharding is enabled for Loki. For information see "Enabling automatic stream sharding".
```

Comment:
It would be nice if we could add the link to "Enabling automatic stream sharding"